clients/venice-consumer/build.gradle

```diff
@@ -3,6 +3,7 @@ evaluationDependsOn(':internal:venice-test-common')
 dependencies {
   implementation project (':internal:venice-common')
   implementation project (':clients:venice-thin-client')
+  implementation project (':clients:da-vinci-client')
```

Comment:
I suppose it is needed by InMemoeryStorageEngine which we use it here as a in-mem cache thing. If we don't do this we might need to move the storage engine layer to venice-server module?

Reply:
oh no that's also not right, venice-server also depends on da-vinci-client

Reply:
You have it right.  The inMemoryStorageEngine is in fact under davinci.

I'm not sure if this is a good or bad thing, but maybe I lack imagination, what're you thinking @FelixGV?

Reply:
Hmm, things are a bit tangled up, which makes me wonder if it's worth having a separate module at all.

If we do want separate modules, then conceptually DVC ought to depend on the consumer, since consumption is a subset of the responsibilities of DVC.

What would be the minimum set of classes that would need to move to the consumer module in order for DVC to depend on consumer and consumer not to depend on DVC? Some (not all I guess?) of the chunking adapter implementations, the storage engine interface, the in-mem storage engine (but not the other ones)... is that it?

Reply:
The consumer module here only contains classes related to change capture and that one div class.  I say we change the name to make it a client and regulate the div to a consumer submodule or common?